packages/ui/src/mocks/data/seedForum.ts

```diff
@@ -36,9 +37,13 @@ export interface RawForumPostMock {
 export function seedForumCategory(forumCategoryData: RawForumCategoryMock, server: any) {
   return server.schema.create('ForumCategory', {
     ...forumCategoryData,
+    moderators: seedModerators(forumCategoryData.moderators, server),
```

Comment:
That shouldn't be necessary, I'll check how to handle that.

Reply:
OK, so for the one-to-many associations you can use ids only. I had to check how it is stored in MirageJS since `s` use there could go both ways. The proper one here is `moderatorIds`.